erratatool.yml

```diff
@@ -62,11 +61,11 @@ boilerplates:
     solution: &common_solution |
       Before applying this update, ensure all previously released errata relevant to your system is applied.
 
-      See the following documentation, which will be updated shortly for release 3.11.z, for important instructions on how to upgrade your cluster and fully apply this asynchronous errata update:
+      For OpenShift Container Platform 3.11 see the following documentation, which will be updated shortly for this release, for important instructions on how to upgrade your cluster and fully apply this asynchronous errata update:
 
       https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html
 
-      This update is available via the Red Hat Network. Details on how to use the Red Hat Network to apply this update are available at https://access.redhat.com/articles/11258
+      Details on how to access this content are available at https://docs.openshift.com/container-platform/3.11/upgrading/index.html
```

Comment:
Shouldn't this be added to boilerplate.image.description as well?

Reply:
I'm always open to further improvements here, but I also think it's fair to have the "here's how you upgrade" instructions attached to the RPM advisory; in 3.11, you can't really use the images until the RPMs have been shipped.